contracts/core/libraries/Payouts.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+library Payouts {
+    struct PayoutSummary {
+        address recipient;
+        uint256 amount;
+        uint256 percentage;
```

Comment:
^ Do we want to standardise this to just use amounts and keep it cleaner. ?

Reply:
Since we just have `distribute` on the interface now, it's up to individual strategies how they implement this. I'll get rid of it.